noxfile.py

```diff
@@ -48,7 +48,7 @@ def default(session):
         # Since many tests are skipped due to missing dependencies, test
         # coverage is much lower in Python 3.8. Remove once we can test with
         # pyarrow.
-        coverage_fail_under = "--cov-fail-under=92"
+        coverage_fail_under = "--cov-fail-under=91"
```

Comment:
Grrr.  We need to get rid of these majyk numbers altogether.  Individual unit test sessions don't need to be enforcing coverage, since we explicitly aggregate them, and require 100% on the aggregate.

Reply:
I'll put that to the agenda of the next client libs meeting.